tasks/startSelenium.js

```diff
@@ -0,0 +1,84 @@
+var _ = require('lodash');
+var request = require('request');
+var fs = require('fs');
+var path = require('path');
+var colors = require('ansicolors');
+var crypto = require('crypto');
+var {spawn} = require('child_process');
+
+module.exports = function (grunt) {
+  grunt.registerTask('startSelenium', 'Start an instance of selenium standalone', function (keepalive) {
+    const done = this.async();
+    const config = _.defaults(this.options(), {
```

Comment:
[`this.options()`](http://gruntjs.com/api/inside-tasks#this.options) actually accepts the defaults as it's first arg